guides/md/keras_nlp/transformer_pretraining.md

```diff
@@ -636,12 +660,329 @@ print(restored_model(inference_data))
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.
```

Comment:
Can we remove this large block of WARNING and INFO logs for now? Not sure what's going on here.